plugins/services/src/js/pages/task-details/__tests__/TaskDetail-test.js

```diff
@@ -57,6 +45,20 @@ describe("TaskDetail", function() {
 
     TaskDirectoryStore.fetchDirectory = jasmine.createSpy("fetchDirectory");
     TaskDirectoryStore.setPath = jasmine.createSpy("setPath");
+
+    this.container = global.document.createElement("div");
```

Comment:
We have to mock the store _before_ creating the instance. Otherwise the component will start listening  for the Store and the store will try to connect.